stdlib/Random/src/Random.jl

```diff
@@ -16,7 +16,7 @@ using Base.GMP: Limb
 import SHA
 
 using Base: BitInteger, BitInteger_types, BitUnsigned, require_one_based_indexing
-
+using Base.Libc
```

Comment:
seems unused? might be good to be precise here though:
```suggestion
using Base.Libc: memcpy
```